typescript/src/entities/product.test.ts

```diff
@@ -29,6 +29,18 @@ test('can fetch product from server', () => {
   return merchi.Product.get(1).then(product => expect(product.name).toBe(testName));
 });
 
+test('can specify options in request', () => {
+  const merchi = new Merchi();
+  const testName = 'S7qHUfV_dr5l';
+  mockFetch(true, {'product': {'name': testName}}, 200);
+  (window as any).merchiBackendUri = 'http://override.example.com/';
+  const options = {includeArchived: true,
+                   withRights: true}; 
+  return merchi.Product.get(1, options).then(product => expect(product.name).toBe(testName));
```

Comment:
this testcase can only ensure when you add  "includeArchived", "withRights" to options does not break get function.

Please write a testcase for backendFetch ensure it fetch(url.toString(), options) been called with correct parameters.

Reply:
my experience of jest is very limited but suppose able to do it by
>
// The first arg of the first call to the function was 'first arg'
expect(tetch.mock.calls[0][0]).toBe('first arg');

Reply:
yes